src/Axe/Models/CodeBlock.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Axe.Models
+{
+    /// <summary>
+    /// Class represents code block which is used in Roslyn platform compiler
+    /// </summary>
+    public class CodeBlock
+    {
+        /// <summary>
+        /// Identifier
```

Comment:
format for properties summary is "Gets or sets %property function%" (or "Gets %property function%" for properties without setter), e.g. "Gets or sets unique identifier"